integration_test/tests/common/mod.rs

```diff
@@ -324,17 +357,25 @@ pub fn fetch_and_get_data(
 }
 
 #[allow(dead_code)]
-///  Helper function for checking resultset metadata
+/// Helper function for checking result set metadata. It optionally takes a list
+/// of attributes to get. If none is provided, the following defaults will be
+/// checked. It also optionally invokes SQLDescribeColW.
 ///  - SQLNumResultCols()
 ///  - For columns 1 to {numCols}
+///      - (Optional) SQLDescribeColW
 ///      - SQLColAttributeW(SQL_DESC_CONCISE_TYPE)
```

Comment:
I think this could be clarified now with the new logic. Even something like:

`SQLColAttributeW for all attrs; if not attrs: [...list the defaults...]`

maybe?